core/api/src/main/java/com/alipay/sofa/rpc/common/RpcOptions.java

```diff
@@ -444,6 +444,10 @@
      * 是否跨接口的长连接复用
      */
     public static final String TRANSPORT_CONNECTION_REUSE         = "transport.connection.reuse";
+    /**
+     * Whether the Http2 Cleartext protocol client uses Prior Knowledge to start Http2
+     */
+    public static final String TRANSPORT_H2C_USE_PRIOR_KNOWLEDGE  = "transport.h2c.use_prior_knowledge";
```

Comment:
change to `transport.client.h2c.usePriorKnowledge`

Reply:
Ok